packages/scripts/scripts/lint-md.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	fromConfigRoot,
+	getArgsFromCLI,
+	hasArgInCLI,
+	hasFileArgInCLI,
+	hasProjectFile,
+	hasPackageProp,
+} = require( '../utils' );
+
+const args = getArgsFromCLI();
+
+const defaultFilesArgs = hasFileArgInCLI() ? [] : [ '**/*.md' ];
+
+// See: https://eslint.org/docs/user-guide/configuring#using-configuration-files-1.
+const hasLintConfig = hasArgInCLI( '-c' ) ||
```

Comment:
It probably could be extracted to a helper function to avoid code duplication, but more importantly, it will help to prevent diverging implementations. 

See:
https://github.com/WordPress/gutenberg/blob/50f471e0596a36d4b6a2df2b9cae382b85dde30b/packages/scripts/utils/config.js#L13-L28

Reply:
This part is no longer needed, because I changed things around, creating a new config just for markdown to allow for overriding some of the rules.

Reply:
One of these days if no one beats me to it is to swap all this out with [cosmiconfig(https://www.npmjs.com/package/cosmiconfig)